scanner/fetcher.go

```diff
@@ -230,7 +231,8 @@ func (f *Fetcher) updateSTH(ctx context.Context) error {
 
 		quick := time.Now().Before(quickDeadline)
 		if sth.TreeSize <= lastSize || quick && sth.TreeSize < targetSize {
-			return errors.New("waiting for bigger STH")
+			// Use an explicitly retriable error code.
```

Comment:
`// TODO: Use RetriableError(fmt.Errorf(...)) instead.`

Reply:
Done, assuming that's now the plan for `backoff`